docs/releasing/releasing.md

```diff
@@ -1,16 +1,17 @@
 # Release generic firmware for Twomes measurement devices
```

Comment:
@henriterhofte This information is very specific to us. Do we need to make this more universal?:
- You need a workflow file (link to it)
- Someone with the permissions to create a release has to publish it.

Reply:
I tried to make it a bit more universal in https://github.com/energietransitie/twomes-generic-esp-firmware/pull/61/commits/9f0fba3142314d61b2eb14cf0c75a22ecd945cb5

I think there might still be an overlap in docs\releasing\releasing.md versus docs\releasing\secure_boot_firmware.md

Can you check whether the sections starting at /docs/releasing/secure_boot_firmware.md#L10 and /docs/releasing/secure_boot_firmware.md#L16 are still needed? It seems the GitHub Actions workflow (https://github.com/energietransitie/twomes-generic-esp-firmware/blob/main/.github/workflows/release.yml which seems also documented in docs\releasing\releasing.md is all we need?